src/scss/misc/_colors.scss

```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```

Comment:
Just talking about colors right now. We dont have to change shapes in order to pick colors. He is so close to a good color palette to use.